File: ios/Sasquatsh/Configuration/AppConfig.swift
import Foundation

enum AppConfig {
    static let supabaseFunctionsURL = "https://yyfukoddeyiaxiufztdx.supabase.co/functions/v1"
    static let supabaseAnonKey = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Inl5ZnVrb2RkZXlpYXhpdWZ6dGR4Iiwicm9sZSI6ImFub24iLCJpYXQiOjE3NzE5MTAxODIsImV4cCI6MjA4NzQ4NjE4Mn0.aujhFFlmiN_rswvYK4-yMrcuiCSa5osg-0i2aINvOYw"
    static let appScheme = "sasquatsh"
    static let webDomain = "sasquatsh.com"
    static let pricingURL = URL(string: "https://\(webDomain)/pricing")!
    static let billingURL = URL(string: "https://\(webDomain)/billing")!
}

File: android/app/src/main/java/com/sasquatsh/app/config/AppConfig.kt
package com.sasquatsh.app.config

import com.sasquatsh.app.BuildConfig

object AppConfig {
    val supabaseFunctionsUrl: String = BuildConfig.SUPABASE_FUNCTIONS_URL
    val supabaseAnonKey: String = BuildConfig.SUPABASE_ANON_KEY
    val googleWebClientId: String = BuildConfig.GOOGLE_WEB_CLIENT_ID
    const val APP_SCHEME = "sasquatsh"
    const val WEB_DOMAIN = "sasquatsh.com"
    const val PRICING_URL = "https://$WEB_DOMAIN/pricing"
    const val BILLING_URL = "https://$WEB_DOMAIN/billing"
}
